i915.c

```diff
@@ -178,53 +178,58 @@ static int i915_add_combinations(struct driver *drv)
 	drv_modify_combination(drv, DRM_FORMAT_R8, &metadata,
 			       BO_USE_CAMERA_READ | BO_USE_CAMERA_WRITE);
 
-	render = unset_flags(render, linear_mask);
-	scanout_and_render = unset_flags(scanout_and_render, linear_mask);
-	/* On ADL-P vm mode on 5.10 kernel, BO_USE_SCANOUT is not well supported for tiled bo */
-	if (is_in_vm() && i915->is_adlp)
-	    scanout_and_render = unset_flags(scanout_and_render, BO_USE_SCANOUT);
-
-	metadata.tiling = I915_TILING_X;
-	metadata.priority = 2;
-	metadata.modifier = I915_FORMAT_MOD_X_TILED;
-
-        // In sriov mode, MMAP_GTT will fail for tiled buffer.
-        if ((drv->gpu_grp_type == TWO_GPU_IGPU_VIRTIO) || (drv->gpu_grp_type == THREE_GPU_IGPU_VIRTIO_DGPU))
-                scanout_and_render =
-                        unset_flags(scanout_and_render, BO_USE_SW_READ_RARELY | BO_USE_SW_WRITE_RARELY);
-
-	drv_add_combinations(drv, render_formats, ARRAY_SIZE(render_formats), &metadata, render);
-	drv_add_combinations(drv, scanout_render_formats, ARRAY_SIZE(scanout_render_formats),
-			     &metadata, scanout_and_render);
-
-	scanout_and_render =
-	    unset_flags(scanout_and_render, BO_USE_SW_READ_RARELY | BO_USE_SW_WRITE_RARELY);
-
-	metadata.tiling = I915_TILING_Y;
-	metadata.priority = 3;
-	metadata.modifier = I915_FORMAT_MOD_Y_TILED;
-
-	// dGPU do not support Tiling Y mode
-	if ((drv->gpu_grp_type == TWO_GPU_IGPU_DGPU) || (drv->gpu_grp_type == THREE_GPU_IGPU_VIRTIO_DGPU))
-		 scanout_and_render = unset_flags(scanout_and_render, BO_USE_SCANOUT);
+	/* Only support tiling none on SR-IOV */
+	if ((drv->gpu_grp_type != TWO_GPU_IGPU_VIRTIO) &&
+	    (drv->gpu_grp_type == THREE_GPU_IGPU_VIRTIO_DGPU)) {
```

Comment:
should be:
	if ((drv->gpu_grp_type != TWO_GPU_IGPU_VIRTIO) &&
	    (drv->gpu_grp_type != THREE_GPU_IGPU_VIRTIO_DGPU)) {
?